src/Delegation.sol

```diff
@@ -358,6 +384,44 @@ contract Delegation is IDelegation, EIP712, GuardedExecutor {
         return true;
     }
 
+    /// @dev Can be used to pause/unpause the contract, in case of emergencies.
+    /// - Pausing the contract will make all signature validations fail,
+    ///   effectively blocking all pay, execute, isValidSignature attempts.
```

Comment:
To be precise, the pausing functionality only affects workflows where `unwrapAndValidateSignature` is used.

The `execute` workflow where the EOA executes itself won't be blocked. This is **ok**, since this workflow is very simple, well-understood, and is mostly based on the audited Solady's 7821.